doc/01_overview/index.rst

```diff
@@ -0,0 +1,16 @@
+Introduction to Ibex
+====================
+
+Ibex is a production-quality open source 32 bit RISC-V CPU core written in SystemVerilog.
+The CPU core is heavily parametrizable and well suited for embedded control applications.
+Ibex is being extensively verified and has seen multiple tape-outs.
```

Comment:
Are there any links to the tape-outs in the doc? Quick search for did not show any. Next sentence says more info, so readers might be inclined to search for a list...

Reply:
No, not yet. I hope to be able to add it as we move along.